src/lib/input/input-container.scss

```diff
@@ -30,3 +30,9 @@ md-input-container {
     }
   }
 }
+
+// Prevents the prefix and suffix from stretching together with the container.
+.md-input-prefix, .md-input-suffix {
+  width: 0.1px;
```

Comment:
I see, is there a reason we have to use table as opposed to flex? I had started a PR to convert it to flex at some point, but I don't know what happened to it...